Marlin/Configuration_adv.h

```diff
@@ -373,6 +373,12 @@
       // without modifying the firmware (through the "M218 T1 X???" command).
       // Remember: you should set the second extruder x-offset to 0 in your slicer.
 
+      /* INDEPENDENT_Z_OFFSETS Allows setting a seperate z offset for X2. This allows compensation
+      * for tool heads that are at different heights. Please note this will be ignored in duplicate mode
+      * and the default z probe offset will be used.
+      */
+  //#define INDEPENDENT_Z_OFFSETS
```

Comment:
- Please follow the established standards for comment formatting and indentation.
- The word "separate" is misspelled.
- "z" should be capitalized.

Admittedly the comments for `DUAL_X_CARRIAGE` are pretty terrible. I will clean up the others.